src/app/rundown-view/components/rundown-tooltips/video-hover-scrub/video-hover-scrub.component.html

```diff
@@ -1,4 +1,4 @@
-<video #videoElementRef [src]="wasVideoVisible ? hoverScrubVideoSource : ''"></video>
+<video #videoElementRef [src]="isVideoVisible ? getVideoPreviewUrl() : ''"></video>
```

Comment:
Is this correct renaming, i just to make sure that it has been given a second thought. `wasVideoVisible` sounds like if the video has been made visible prior to this (like if we have the element reference from before). In that case it could be named accordingly, if not i apologize for the bother.

Reply:
`wasVideoVisible` indicates for me that we used to have the video visible, but now it no longer is, since `was` is past tense.

Reply:
The attribute is now completely gone.